formatter/fmt

```diff
@@ -0,0 +1,47 @@
+#! /usr/bin/env bash
+
+set -e
+set +x
+set -o pipefail
+
+cd /code
+
+# Gets the list of file paths for files that are tracked by git and have
+# changes relative to the main branch.
+function get_tracked_modified_files() {
+  git diff --name-only main
```

Comment:
This lists what I assume is all the files for me... Could this be something that won't work for everyones setup?

Reply:
oh hmm, what's your local git state? are you on main? a branch?

Reply:
Oh, I was on a renovate bot branch.. I made a new branch and it is WAI showing no files.